src/components/slop-fest-modal.jsx

```diff
@@ -0,0 +1,189 @@
+            },
+            creator: {
+              connect: { id: currentUser.id },
+            },
+          },
+        },
+      }).then(() => closeModal("create"));
+    } catch (error) {
+      console.log(`Error: ${error.message}`);
+    }
+  };
+
+  return (
+    <Modal title="OH WE FESTIN">
+      <Form
+        onSubmit={handleSubmit(onSubmit)}
+        className={"flex flex-col items-center p-6"}
+      >
+        <div className="flex flex-col">
+          <div className="flex-col">
+            <Form.TextInput
+              labelText="Name it!"
+              className={`w-[373px]`}
+              id={"name"}
+              register={register("name", {
+                required: true,
+                pattern: {
+                  value: /^\S/,
+                  message: "Must not start with a space",
+                },
+              })}
+              onChange={(evt) => {
+                setValue("name", evt.target.value, { shouldValidate: true });
+              }}
+              isValid={!errors.name}
+            />
+            {errors.name ? (
+              <Form.Feedback
+                message={"Must contain more than one character"}
+                className={""}
+              />
+            ) : (
+              ""
+            )}
+          </div>
+          <div className="flex flex-row justify-between">
+            <div className="flex-col">
+              <Form.DateDropdown
+                labelText={"Start Date"}
+                className={"w-[176px]"}
+                required
+                name="startDate"
+                isValid={!errors.startDate}
+                date={startDate}
+                register={register("startDate", {
+                  required: "A start date is required",
+                  valueAsDate: true,
+                })}
+                onChange={(date) => {
+                  setStartDate(date);
+                  setValue("startDate", date, {
+                    shouldValidate: true,
+                  });
+                }}
+              />
+              {errors.startDate ? (
+                <Form.Feedback message={"Must be a valid date"} />
+              ) : (
+                ""
+              )}
+            </div>
+            <div className="flex-col">
+              <Form.DateDropdown
+                labelText={"End Date"}
+                className={"w-[176px] h-12"}
+                required
+                name="endDate"
+                isValid={!errors.endDate}
+                date={endDate}
+                onChange={(date) => {
+                  setEndDate(date);
+                  setValue("endDate", date, {
+                    shouldValidate: true,
+                  });
+                }}
+              />
+              {errors.endDate ? (
+                <Form.Feedback message={"Must be a valid date"} />
+              ) : (
+                ""
+              )}
+            </div>
+          </div>
+          <Form.Combobox
```

Comment:
This is a weird behavior I noticed. The combo-box works great! 

When I open it up, I get a default value of the user who is creating the fest. I made a comment above about this. I can add additional users and the combo box expands as it should. When I click on the combo box additional options show up. 
![Screenshot 2024-02-27 at 5 00 05 PM](https://github.com/jahorwitz/slopopedia/assets/109189846/adee70ce-b06a-4035-abcb-477ac161e6b1)

The bug appears when I remove myself as a user. If I click back into the combo box, all users available show up, but my own username does not show up. The only way my user shows up again is if I close and re-open the modal.

![Screenshot 2024-02-27 at 5 00 48 PM](https://github.com/jahorwitz/slopopedia/assets/109189846/956e6e21-81a1-4218-8599-793ecd80e261)

Reply:
This has been fixed. The user does not show up on the drop down, but it is automatically added when you submit.